docs/proposals/20210720-use-kcp.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Proposal Template
+authors:
+  - "@charleszheng44"
+  - "@christopherhein"
+reviewers:
+  - "@Fei-Guo"
+  - "@vincepri"
+creation-date: 2021-07-14
+last-updated: 2021-07-14
+status: provisional
+see-also:
+  - "/docs/proposals/20201026-creating-control-plane-components.md"
+  - "/docs/proposals/20210126-nc-and-ncp.md"
+replaces:
+  - "/docs/proposals/20210126-nc-and-ncp.md"
+---
+
+# Using KubeadmControlPlane Provider For CAPN
+
+
+## Table of Contents
+
+<!-- TODO generate TOC -->
+A table of contents is helpful for quickly jumping to sections of a proposal and for highlighting
+any additional information provided beyond the standard proposal template.
+[Tools for generating](https://github.com/ekalinin/github-markdown-toc) a table of contents from markdown are available.
+
+## Glossary
+
+Refer to the [Cluster API Book Glossary](https://sigs.k8s.io/cluster-api-provider-nested/docs/proposals/00_capn-glossary.md).
+
+If this proposal adds new terms, or defines some, make the changes to the book's glossary when in PR stage.
+
+## Summary
+
+Today, with CAPN when you provision your Pod based control plane we have implemented a completely bespoke `ControlPlaneProvider` called the `NestedControlPlane`. The `NestedControlPlane` uses a set of three "nested components" each defining a single component of the control plane, `kube-apiserver` as `NestedAPIServer`, `kube-controller-manager` as `NestedControllerManager` and `etcd` as `NestedEtcd` these allow each component to design it's own spec. Having three nested components makes the CAPI provider bespoke and difficult for a new user that has CAPI experience to translate their existing cluster specs into a nested architecture.
+
+After spending time evaluating CAPI's in-tree `KubeadmControlPlaneProvider` we think this provider can give us the base components necessary using the standardized KCP spec to generate the necessary pod objects. 
```

Comment:
It would be great to add a `KCP` link here, I misunderstanding this KCP as https://github.com/kcp-dev/kcp

Reply:
Good point. Have replaced the `KCP` with `KubeadmControlPlane`.